src/PrizePool.sol

```diff
@@ -479,7 +479,7 @@ contract PrizePool {
             prizeToken.transfer(_feeRecipient, _fee);
```

Comment:
Is it better to just put the payout calculation from line 467 into like 477 rather than creating a variable that's only used once?

Reply:
It's used twice- in the event below too